upload.py

```diff
@@ -628,8 +628,10 @@ def format_option_strings(self, option):
                  dest="save_cookies", default=True,
                  help="Do not save authentication cookies to local disk.")
 group.add_option("--oauth2", action="store_true",
-                 dest="use_oauth2", default=False,
+                 dest="use_oauth2", default=True,
                  help="Use OAuth 2.0 instead of a password.")
+group.add_option("--no_oauth2", action="store_false",
+                 dest="use_oauth2", help="Use password instead of OAuth 2.0.")
```

Comment:
Maybe let's add a comment here, that this is only useful for development purposes.

Reply:
d30742ca79c4ca8dedcf4fbe468e91bd2a3b712a

Reply:
f51d03f8b280c52a6ff0ef8cd5233b6082718ca5